fixtures/contest/led-gun.json

```diff
@@ -0,0 +1,301 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED-GUN",
+  "categories": ["Flower", "Color Changer"],
+  "meta": {
+    "authors": ["Axel"],
+    "createDate": "2024-01-24",
+    "lastModifyDate": "2024-01-24"
+  },
+  "links": {
+    "manual": [
+      "https://contest-lighting.com/downloads/LED-GUN_manuel-v4.0-web-en.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [271, 176, 213],
+    "weight": 4,
+    "power": 25.3,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED Tri"
+    }
+  },
+  "availableChannels": {
+    "Channel 1": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 14],
+          "type": "NoFunction",
+          "comment": "OFF"
+        },
+        {
+          "dmxRange": [15, 124],
+          "type": "Rotation",
+          "speedStart": "slow CW",
+          "speedEnd": "fast CW",
+          "comment": "Fast to Slow"
+        },
+        {
+          "dmxRange": [125, 139],
+          "type": "Rotation",
+          "speed": "stop",
+          "comment": "No rotation"
+        },
+        {
+          "dmxRange": [140, 249],
+          "type": "Rotation",
+          "speedStart": "slow CCW",
+          "speedEnd": "fast CCW",
+          "comment": "Fast to Slow"
+        },
+        {
+          "dmxRange": [250, 255],
+          "type": "SoundSensitivity",
+          "soundSensitivityStart": "low",
+          "soundSensitivityEnd": "high"
+        }
+      ]
+    },
+    "Dimmer / Strobe": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 7],
+          "type": "Intensity",
+          "brightness": "off"
+        },
+        {
+          "dmxRange": [8, 199],
+          "type": "Intensity",
+          "brightnessStart": "0%",
+          "brightnessEnd": "100%"
+        },
+        {
+          "dmxRange": [200, 247],
+          "type": "StrobeSpeed",
+          "speedStart": "slow",
+          "speedEnd": "fast"
```

Comment:
Needs a ShutterStrobe to enable the strobe, in order for speeds to work like this:

```
  "type": "ShutterStrobe",
  "shutterEffect": "Strobe",
```